pypdf/_reader.py

```diff
@@ -999,6 +998,41 @@ def _rebuild_xref_table(self, stream: StreamType) -> None:
             if generation not in self.xref:
                 self.xref[generation] = {}
             self.xref[generation][idnum] = m.start(1)
+
+        logger_warning("parsing for Object Streams", __name__)
```

Comment:
How does this justify a warning?

Reply:
this is just a warning message when parsing/rebuilding the objects tables this helps to follow the process it appears after the message rebuilding the xref table